src/test/java/org/lisoft/lsml/model/item/AmmoWeaponTest.java

```diff
@@ -104,4 +104,30 @@ public final void testSpreadQuirks() throws Exception {
 
         assertEquals(normal * 2, quirked, 0.0);
     }
+    
+    @Test
+    public void testGetVolleySize() throws Exception {
+        final AmmoWeapon ac20 = (AmmoWeapon) ItemDB.lookup("AC/20");
+        final AmmoWeapon lrm10 = (AmmoWeapon) ItemDB.lookup("LRM 10");
+        final AmmoWeapon clrm10 = (AmmoWeapon) ItemDB.lookup("C-LRM 10");
+        final AmmoWeapon cuac10 = (AmmoWeapon) ItemDB.lookup("C-ULTRA AC/10");
+        
+        assertEquals(10.0, lrm10.getVolleySize(),0.0);
```

Comment:
`getVolleySize()` returns an `int` this form of `assertEquals` coerces it to `double` and then compares. Please use `assertEquals(int expected, int actual)` instead of `assertEquals(double expected, double actual, double tolerance)`.

Reply:
Sorry, this one is an artifact of my flopping around on how to get this volley size, num rounds, rounds per shot sorted. I'll fix this to be just the int version.

Reply:
done